share/functions/fish_vi_key_bindings.fish

```diff
@@ -1,3 +1,77 @@
+alias vi_dec 'vi_inc_dec dec'
+alias vi_inc 'vi_inc_dec inc'
+
+# TODO: Currently we do not support hexadecimal and octal values.
+function vi_inc_dec --description 'increment or decrement a number'
+  # The cursor is zero based, but all string functions assume 1 to be
+  # the lowest index. Adjust accordingly.
+  set cursor (math (commandline --cursor) + 1)
+  set line (commandline --current-buffer)
+
+  set candidate (string sub --start $cursor -- $line)
+  if set just_found (string match --ignore-case --regex '^-?[0-9]+' -- $candidate)
```

Comment:
we'll want to declare `set -l just_found` above this line, so both branches always use this local

Reply:
Seems fine to just add `--local` there, no need to move?